wpilibj/src/main/java/edu/wpi/first/wpilibj/RobotDrive.java

```diff
@@ -21,7 +21,7 @@
  * function (intended for hand created drive code, such as autonomous) or with the Tank/Arcade
  * functions intended to be used for Operator Control driving.
  *
- * @deprecated Use DifferentialDrive or MecanumDrive classes instead.
+ * @deprecated Use {@link edu.wpi.first.wpilibj.drive.DifferentialDrive} or {@link edu.wpi.first.wpilibj.drive.MecanumDrive} classes instead.
```

Comment:
Maximum line length is 100 characters.  This needs to be wrapped as it's now ~140 characters.

Reply:
I made a new commit with multiple lines under 100 chars